dbt/adapters/bigquery/connections.py

```diff
@@ -186,12 +186,9 @@ def _connection_keys(self):
             "job_creation_timeout_seconds",
```

Comment:
can't comment further up on linen 174 - but it'd probably be worth a comment there [linking to the docs on this method](https://docs.getdbt.com/guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter#editing-the-connection-manager), and indicating that any elements in the returned set are included in debug stdout so secrets should be omitted.